docs/plugins.md

```diff
@@ -166,4 +166,4 @@ plugins please report them to the corresponding plugin developer.
 
 * [protractor-numerator](https://github.com/Marketionist/protractor-numerator): This plugin gives you readable functions for getting elements by their numbers inside Protractor tests. Adds functions like `.second()`, `.third()`, etc. instead of `.get(1)`, `.get(2)`, etc.
 
-* [Ng-apimock](https://github.com/mdasberg/ng-apimock): this plugin adds the ability to use scenario based api mocking for local development and protractor testing for both AngularJS 1 and 2 applications.
\ No newline at end of file
+* [Ng-apimock](https://github.com/mdasberg/ng-apimock): this plugin adds the ability to use scenario based api mocking for local development and protractor testing for both AngularJS 1 and 2 applications.
```

Comment:
While we are at it Could we change this line to say AngularJS and Angular applications?